packages/bot-skeleton/src/scratch/hooks/block_svg.js

```diff
@@ -269,7 +269,19 @@ Blockly.BlockSvg.prototype.setCollapsed = function (collapsed) {
             this.setCollapsed(false)
         );
 
-        this.appendDummyInput(COLLAPSED_INPUT_NAME).appendField(field_label).appendField(field_expand_icon).init();
+        if (this.type === 'procedures_defreturn' || this.type === 'procedures_defnoreturn') {
```

Comment:
On bot-skeleton we have 5 different function definition I think it's better if you use this.category instead of type 
```suggestion
        if (this.category_ ==='custom_functions') {
```

Reply:
hey, i've checked it with Anton, he said it should be applicable for only these 2 types, other 3 should not be affected